.github/workflows/playwright.yml

```diff
@@ -1,22 +1,53 @@
 name: Playwright Tests
+
 on:
   workflow_call:
     inputs:
       frontendHostname:
         type: string
         description: 'Hostname of the frontend server (optional)'
         required: false
+      playwright_branch:  
+        type: string
+        description: 'The name of the branch to be tested with Playwright.'
+        required: false  
   workflow_dispatch:
     inputs:
       frontendHostname:
         type: string
         description: 'Hostname of the frontend server (optional)'
-        required: false
+        required: true
+      playwright_branch:  
+        type: string
+        description: 'The name of the branch to be tested with Playwright.'
+        required: true 
```

Comment:
I think it would be better to make the parameter optional and use main as a default for backwards compatibility